octodns/provider/ns1.py

```diff
@@ -350,15 +350,18 @@ def _FILTER_CHAIN_WITH_REGION_AND_COUNTRY(self, with_disabled):
         'AS': ('ASIAPAC',),
         'EU': ('EUROPE',),
         'SA': ('SOUTH-AMERICA',),
-        # TODO: what about CA, MX, and all the other NA countries?
-        'NA': ('US-CENTRAL', 'US-EAST', 'US-WEST'),
```

Comment:
Will `_REGION_TO_CONTINENT` above still need it's `US-*` cases to handle the upgrade path? I believe that's the case. Probably makes sense to add a comment up there to that effect to explain why they need to stick around, for now.

Reply:
Yes, thats why I left this block untouched:
```
    _REGION_TO_CONTINENT = {
        'AFRICA': 'AF',
        'ASIAPAC': 'AS',
        'EUROPE': 'EU',
        'SOUTH-AMERICA': 'SA',
        'US-CENTRAL': 'NA',
        'US-EAST': 'NA',
        'US-WEST': 'NA',
    }
```
Adding a comment.

Reply:
Done